.github/CONTRIBUTING.md

```diff
@@ -15,45 +27,50 @@ git clone git@github.com:lightspeed/flame.git
 To create a brand new component, use the following command that will create a base skeleton ready for development:
 
 ```sh
-yarn generate Component
+yarn generate
 ```
 
-Where `Component` is the name of the component you want to create. This will create the base component structure and story for development/documentation in `packages/flame-{component-name}`, which are considered [add-ons / experimental](https://github.com/lightspeed/flame#add-ons-and-experimental-components) until being deemed ready to go into the [mainline package](https://github.com/lightspeed/flame/tree/master/packages/flame).
+You will be given a choice of "Mainline" or "Add-on" component, then have to choose a name for the component.
+
+#### Mainline component
+
+Mainline components will be created under `packages/flame/src/{ComponentName}`, where all other components are located. This will create the base component structure ready for development, including README for docs, story for Storybook development and visual regression test, tests and TypeScript component file.
```

Comment:
Usually, when we create a comma separated list, we aim to have three separate items that are fairly similar in length. The flow helps with readability. Consider the following suggestion:

```suggestion
Mainline components will be created under `packages/flame/src/{ComponentName}`, where all other components are located. The base component structure includes a storybook file, a test file, and a component file.
```